src/vue/components/inflections-table-wide.vue

```diff
@@ -136,39 +154,68 @@ export default {
   },
 
   computed: {
+    title: function () {
+      return this.view.title || this.standardFormData.title || ''
```

Comment:
won't this fail if this.view is null?

Reply:
If the `view` prop is not provided to the component, it will be assigned `fasle` because this is a default value. Then the `view` will hold the value of type `Boolean` which is an object. Because of this, `view.title` will not throw an error, but be undefined.

To test in in browser's console:
```
let a = false
typeof a // returns "boolean"
a.test // returns undefined
```

This allows to avoid additional checks and write it in a simpler way, but if it impacts readability I can add additional checks. What do you think?